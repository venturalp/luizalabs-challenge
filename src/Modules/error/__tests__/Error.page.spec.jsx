import { render } from '@testing-library/react'
import { ThemeWrapper } from 'Commons/tests/Tests.Helpers'
import React from 'react'
import { ErrorPage } from '../Error.page'

describe('test Error page', () => {
  it('render error page', () => {
    const { asFragment } = render(
      <ThemeWrapper>
        <ErrorPage />
      </ThemeWrapper>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
