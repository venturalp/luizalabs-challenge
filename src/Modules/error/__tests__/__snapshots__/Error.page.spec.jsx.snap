// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Error page render error page 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ kyBdbW"
  >
    <h1>
      Page not found
    </h1>
  </div>
</DocumentFragment>
`;
